Project/Sources/Methods/test_method_sblaser.4dm

```diff
@@ -0,0 +1,38 @@
+//%attributes = {}
```

Comment:
very good practice to provide a test method in pull request

maybe prefix it using "_", I see other test method in this project like that

for a test it will be good to have some `ASSERT` :
- like clientId and clientSecret not defined by editing the test (maybe latter a not commited file could provide this info)
- stop if `File("..//private.pem")` is missing and assert that a pem is needed instead of `getText` issue
- ...